docs/how-to/build.md

```diff
@@ -1185,7 +1185,7 @@ Android NNAPI Execution Provider can be built using building commands in [Androi
 
 * Code Signing
 
-  If the development team ID which has a valid code signing certificate is specified, Xcode will code sign the onnxruntime library in the building process, otherwise, the onnxruntime will be built without code signing. It may be required or desired to code sign the library for iOS devices. For more information, see [Code Signing](https://developer.apple.com/support/code-signing/).
+  If the development team ID and/or the code sign identity which has a valid code signing certificate is specified, Xcode will code sign the onnxruntime library in the building process, otherwise, the onnxruntime will be built without code signing. It may be required or desired to code sign the library for iOS devices. For more information, see [Code Signing](https://developer.apple.com/support/code-signing/).
```

Comment:
nits: 
First sentence could be broken into two at the 'otherwise'. 
Is it 'and/or' or 'or'? 
Wording is a little confusing too. Would something like this work:

> If the code signing development team ID or code signing identity is specified, and has a valid code signing certificate, Xcode will code sign the onnxruntime library in the building process. Otherwise, the onnxruntime will be built without code signing.

Reply:
You can specify both team id and signing identity (and they match).
Yes, this is confusing, change and/or -> or